.github/workflows/vm-migration.yml

```diff
@@ -0,0 +1,69 @@
+name: VM migration test
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/vm-migration.yml'
+      - '.github/workflows/testflinger/vm-live-migration-job.yml'
+      - 'tests/vm-migration'
+  schedule:
+    - cron: '38 6 */5 * *'
+  workflow_dispatch:
+    inputs:
+      ubuntu-releases:
+        description: Ubuntu releases to run the tests against. In JSON format, i.e. '["jammy", "noble"]'.
+        type: choice
+        default: '["noble"]'
+        options:
+          - '["jammy", "noble"]'
+          - '["jammy"]'
+          - '["noble"]'
+      snap-track:
+        description: Snap track to run the tests i.e. '["latest/edge", "5.0/candidate"]'.
+        type: choice
+        default: '["latest/edge"]'
+        options:
+          - '["latest/edge"]'
+          - '["latest/candidate"]'
+          - '["latest/stable"]'
+          - '["6/edge"]'
+          - '["6/candidate"]'
+          - '["6/stable"]'
+          - '["5.21/edge"]'
+          - '["5.21/candidate"]'
+          - '["5.21/stable"]'
+
+jobs:
+  vm-migration:
+    runs-on: [self-hosted, self-hosted-linux-amd64-jammy-private-endpoint-medium]
+    strategy:
+      matrix:
+        track: ${{ fromJSON(inputs.snap-track || '["latest/edge"]') }}
+        os:
+          - core24-latest
```

Comment:
I think this means to test on Ubuntu Core which is potentially not what we want here.